fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -156,16 +162,14 @@ async def close(self, code: int = 1000):
             # Case opened, we have something to close.
             await self._websocket.close(code)
 
-def isNotInvalidStatusCode(value):
-    return not isinstance(value, InvalidStatusCode)
-
 
 def isNotForbbiden(value) -> bool:
     """
     Returns:
-        bool: Returns True as long as the given exception value is not InvalidStatusCode with 401 or 403
+        bool: Returns True as long as the given exception value doesn't hold HTTP status codes 401 or 403
     """
-    return not (isinstance(value, InvalidStatusCode) and (value.status_code == 401 or value.status_code == 403))
+    value = getattr(value, "response", value)  # `websockets.InvalidStatus` exception contains a status code inside the `response` property
+    return not (hasattr(value, "status_code") and value.status_code in (401, 403))
```

Comment:
Combines `websockets.InvalidStatus`, `websockets.InvalidStatusCode` and `websocket.WebSocketBadStatusException` exceptions

Reply:
Hi @mberdyshev - sorry for the delay here - we'll clear time for it next week

Reply:
To push the error further on the stack, I guess. With `return` it would be just a line of the text that probably might be unintentionally missed.
As I understand the handlers are designed for internal use only, inside `WebSocketRpcClient`. So it shouldn't matter much, I think.